airbyte-integrations/connectors/source-googleanalytics-singer/googleanalytics_singer_source/source.py

```diff
@@ -0,0 +1,65 @@
+"""
+MIT License
+
+Copyright (c) 2020 Airbyte
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import json
+import os
+import tempfile
+
+from airbyte_protocol import AirbyteConnectionStatus, Status
+from base_singer import SingerSource
+from tap_google_analytics import GAClient
+
+CREDENTIALS_FILE = os.path.join(tempfile.gettempdir(), "credentials.json")
+
+
+class GoogleAnalyticsSingerSource(SingerSource):
+    def __init__(self):
+        pass
+
+    def check(self, logger, config_container) -> AirbyteConnectionStatus:
+        try:
+            client_secrets = json.loads(config_container.raw_config["credentials_json"])
+            additional_fields = {"end_date": "2020-10-01T00:00:00Z", "client_secrets": client_secrets}
```

Comment:
is there any danger of this date being "wrong"? just asking since it's in the past